dags/analyzer_helper/tests/integration/test_discord_extract_raw_info.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from datetime import datetime
+
+from analyzer_helper.discord.discord_extract_raw_infos import DiscordExtractRawInfos
+from bson import ObjectId
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestDiscordExtractRawInfos(unittest.TestCase):
+    def setUp(self):
+        self.client = MongoSingleton.get_instance().client
+        self.guild_id = "discord_guild_id"
+        self.platform_id = "platform_db"
+        self.guild_db = self.client[self.guild_id]
+        self.platform_db = self.client[self.platform_id]
+        self.rawinfos_collection = self.guild_db["rawinfos"]
+        self.rawinfos_collection.delete_many({})
```

Comment:
We ideally want to clear the `rawmemberactivities` collection because in case of recompute equal to False, we would check its collection data.